pyproject.toml

```diff
@@ -54,7 +54,8 @@ where = ["src"]
 include = ["instructlab", "instructlab.chat", "instructlab.generator", "instructlab.train", "instructlab.train.lora_mlx", "instructlab.train.lora_mlx.models", "instructlab.llamacpp", "instructlab.mlx_explore", "instructlab.schema.v1", "instructlab.schema.v2"]
 
 [tool.setuptools.package-data]
-"instructlab.llamacpp" = ["quantize"]
+"instructlab.llamacpp.macos" = ["quantize"]
+"instructlab.llamacpp.linux" = ["quantize"]
```

Comment:
The quantize binary is a ARM64 macOS program and does not work on Linux

```shell
$ file src/instructlab/llamacpp/quantize 
src/instructlab/llamacpp/quantize: Mach-O 64-bit arm64 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK|PIE>
```

Reply:
i have two, one built on linux one built on macos. They now live in `/llamacpp/linux/quantize` and `/llamacpp/macos/quantize`.

This needs a better solution but I believe is outside the scope of this PR

Reply:
thanks for catching this. Will build a linux one now